### DISCUSSION TITLE
Full Gesture Support

### DISCUSSION THREAD

**Discussion by CreativeCodeCat** [iamrasel/lunar-launcher#41] [Ideas]
he if i was to work on it would you be interested in having full support for gestures for example `up down left right` that can open app, open power menu, lock screen, open quick setting, open notification draw, open app draw, show recents, take screenshot or disable

**Comment by iamrasel** [comment]
Currently available gestures are: **up** - quick access dialog, **down** - notification panel, **left** - app drawer/list, **right** - feeds/widgets. These actions are all necessary, so there's no way to replace them (except, swipe down gesture).
